qa/tasks/cephfs/filesystem.py

```diff
@@ -778,7 +778,9 @@ def get_daemon_names(self, state=None, status=None):
         """
         mdsmap = self.get_mds_map(status)
         result = []
-        for mds_status in sorted(mdsmap['info'].values(), lambda a, b: cmp(a['rank'], b['rank'])):
+        for mds_status in sorted(
+                mdsmap['info'].values(),
+                lambda a, b: (a['rank'] > b['rank']) - (a['rank'] < b['rank'])):
```

Comment:
Why not just use key?
```
for mds_status in sorted(mdsmap['info'].values(), key=lambda _: _['rank']):
```

Reply:
I just followed the python docs how to replace `cmp`. See https://docs.python.org/3.0/whatsnew/3.0.html#ordering-comparisons

Reply:
@toabctl I understand, and one of the item from this guide says: **Use the key argument instead.**, see the cite:
```
builtin.sorted() and list.sort() no longer accept the cmp argument 
providing a comparison function. Use the key argument instead. 
N.B. the key and reverse arguments are now “keyword-only”.
```

Reply:
Another cite:
```
If you really need the cmp() functionality, you could use the expression (a > b) - (a < b) as the equivalent for cmp(a, b).
```
And given that I want to change as less as possible I decided to go that way.

Reply:
@toabctl yes, but the cmp is not required here, IMHO.

Reply:
I suggest to do things right way from with on PR instead of having a bunch of overlapping commits.

Reply:
No longer part of this PR so marking this as resolved